### PR TITLE
Upgrade to cordova-android@10

### PR DESCRIPTION
**Pull request by renanccastro**
Required changes
- use WebViewAssetLoader instead of the local port mechanism for intercepting URLs (webapp plugin update)
- Change usage of cordova-android api, it's not included anymore in the generated project
- AndroidXEnabled is now true
- Allow clear text Traffic(http) on android for development purposes

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
Looks like `tools/tests/cordova-builds.js` will need some review as well.

File: packages/webapp/package.js [hunk 1]
```diff
@@ -20,9 +20,9 @@ Npm.strip({
   useragent: ["test/"]
 });
 
+// whitelist plugin is now included in the core
```
Comment (StorytellerCZ) [inline packages/webapp/package.js:23]: Nice comment about the change, but that should have been just a comment here, no need to have it in the code itself.

File: tools/cordova/builder.js [hunk 2]
```diff
@@ -115,7 +115,7 @@ export class CordovaBuilder {
       author: 'A Meteor Developer',
       email: 'n/a',
       website: 'n/a',
-      contentUrl: `http://localhost:${cordovaServerPort}/`
```
Comment (StorytellerCZ) [inline tools/cordova/builder.js:118]: I'll need a bit of context on this removal.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]
Finish TODO, other than that it looks pretty good.

File: tools/cordova/builder.js (hunk 2 above)
Comment (renanccastro) [reply tools/cordova/builder.js]: I've rolled back, it's actually not necessary and would introduce issues on multiple apps on iOS

**Review by StorytellerCZ — APPROVED** [review]

**Review by filipenevola — APPROVED** [review]

**Pull request merged**